src/data/auth.ts

```diff
@@ -137,7 +137,13 @@ export const adminChangePassword = (
     password,
   });
 
-export const deleteAllRefreshTokens = (hass: HomeAssistant) =>
+export const deleteAllRefreshTokens = (
+  hass: HomeAssistant,
+  token_type?: string,
```

Comment:
This could use a little more typing: 

```suggestion
  token_type?: RefreshTokenType,
```

Since the type doesn't exist, in the type RefreshToken following changes are needed: 
```ts
export type RefreshTokenType = "normal" | "long_lived_access_token";

export interface RefreshToken {
  client_icon?: string;
  client_id: string;
  client_name?: string;
  created_at: string;
  id: string;
  is_current: boolean;
  last_used_at?: string;
  last_used_ip?: string;
  type: RefreshTokenType;
}

```